lineman/app/components/group_page/group_page_controller.coffee

```diff
@@ -8,9 +8,15 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $rout
     $rootScope.$broadcast 'setTitle', @group.fullName()
     $rootScope.$broadcast 'analyticsSetGroup', @group
     MessageChannelService.subscribeTo("/group-#{@group.key}")
+    @setPhotos()
   , (error) ->
     $rootScope.$broadcast('pageError', error)
 
+  @setPhotos = =>
```

Comment:
Intended workflow is: 
- On pageload, set the photo styles
- As a callback from a photo being uploaded, reset the styles
(I couldn't quite get a simple two-way binding to work for me here)